text/0000-serialization-format.md

```diff
@@ -0,0 +1,95 @@
+# Serialization Format
+
+## Summary
+
+[summary]: #summary
+
+Definition of a versioned, backwards-compatible and extensible serialization format for cartographer-maps.
+
+This RFC is a small deviation of the previously proposed header format from [RFC-0015](https://gitub.com/googlecartographer/rfc/text/0015-serialization-header.md).
+
+## Motivation
+
+[motivation]: #motivation
+
+At the moment, the file format for serialized *PoseGraphs* is defined purely within the serialization & deserialization routines of cartographer. Since we serialize all information into a binary stream of `protobuf` messages (*pbstream*) without a format-version information, extending or changing the format inevitably leads to a breaking change for cartographer users, as there is no way of falling back to load previously serialized maps.
```

Comment:
I am not sure that it is completely correct to say that we have a format for "serialized PoseGraph". We are trying to serialize the whole state of Cartographer. Now it is mostly the pose graph and trajectory/sensor options/data.

Reply:
Rephrased.